package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Weeb bot",
   "main": "index.js",
   "scripts": {
-    "start": "nodemon index.js",
+    "dev": "nodemon index.js",
```

Comment:
we will still need npm start, can you make dev a separate key from start?

Reply:
what will start be for? "node index.js" in prod?

Reply:
All node projects need an entry point, so we should leave it alone and add the `npm run dev` as a separate key